2024/planning-process.md

```diff
@@ -0,0 +1,132 @@
+        5. [WHATWG](https://whatwg.org/)
+    3. Describe the benefit to web developers and the web itself
+    4. Link to existing tests (if any) and/or describe plan to create needed tests
+5. Investigation Project proposals must:
+    1. Outline their scope and their goals,
+    2. Not substantially be work better suited to a group within a standards development organization,
+    3. Clearly provide present or future impact on cross-browser interoperability.
+
+As soon as each proposal is submitted as a GitHub issue, discussion of that proposal can begin. Questions can be asked. The proposal can be edited and refined.
+
+At the end of the three weeks, the deadline will end the period when brand new proposals can be submitted.
+
+A fourth week will allow for continued discussion and refinement of already-submitted proposals. The deadline at the end of the fourth week ends the opportunity to significantly change the scope of proposals.  The proposals will be evaluated on their state at that time.
+
+
+## Focus Area Selection
+
+### Elimination Round 1: Disqualification
+
+After the Call of Proposals Period has ended, the Interop Team will finalize an official list of all Focus Area proposals.
+
+Any Focus Area proposal that does not meet the minimum publicly-stated selection criteria will be considered for preliminary elimination. For example:
+
+1. The proposal is overly vague or broad.
+2. The proposed technology is already interoperable according to WPT results.
+3. The proposed technology is not on a standards track, or its standard is not sufficiently mature.
+
+Decisions will be made at a specific meeting for discussion of preliminary eliminations. The minutes of this meeting are confidential to the team. Any member organization can bring up a proposal they believe should be eliminated. Brief debate can be had. If **there is quick consensus**, that proposal will be eliminated. If there is not consensus and a longer debate is needed, the proposal will be not be eliminated and will instead move on to the next round.
+
+1. Setup *Disqualification meeting*
+2. Each organization can propose items for disqualification.
+3. Brief debate on each proposal:
+    1. if there is quick consensus, disqualify the proposal
+    2. otherwise, send the proposal to Round 2
+4. Publish the list of proposals pushed to Round 2.
+
+### Elimination Round 2: Reduction
+
+At this point, there is a clear list of Focus Area proposals under consideration. Each organization has two weeks to review the list of proposals and make Round 2 decisions.
+
+Each member organization chooses 40 proposals they would like to see move on to Round 3; these selections are confidential to the team. There is no ranking or scoring at this point, a simple “yes, we want this proposal to stay in consideration”. Once all the “yeses” are gathered, any proposal that does not have the support of at least one organization is eliminated.
```

Comment:
Regarding the confidentiality wording added in [7093980](https://github.com/web-platform-tests/interop/pull/390/commits/70939802be67b4354a0436271cdf13b923bdcc1d), there's a proposal I'd floated a couple meetings ago that I'd like to revisit in the interest of nudging the process towards openness where possible:

> Once the announcement of selected proposals has been made, members may choose to share publicly the anonymized, cumulative results of each round. Specifically, that is: the set of proposals that were disqualified in round 1, the number of "yeses" that each proposal received in round 2, and the number of points that each proposal received in round 3.

This would maintain the confidentiality of individual member organizations' positions, which I hope addresses concerns raised about positions being taken out of context.

There was also a question raised about whether this is worth the trouble for the Interop team to report. I don't think there's much work involved since we will need to compute these totals anyway as part of the process, but nevertheless I think that concern is addressed by making the sharing of this info an option for members rather than an obligation of the Interop team.

Thoughts?